MediaBrowser.Api/Playback/Progressive/BaseProgressiveStreamingService.cs

```diff
@@ -324,7 +324,8 @@ private async Task<object> GetStaticRemoteStreamResult(StreamState state, Dictio
             // Seeing cases of -1 here
             if (response.ContentLength.HasValue && response.ContentLength.Value >= 0)
             {
-                responseHeaders["Content-Length"] = response.ContentLength.Value.ToString(UsCulture);
+                // TODO
```

Comment:
pls explain the TODO

Reply:
Old commit, but I explained it further up